pywinauto/controls/common_controls.py

```diff
@@ -674,15 +674,27 @@ def __init__(self, hwnd):
 
         if self.is_unicode():
             self.create_buffer = ctypes.create_unicode_buffer
-            self.LVCOLUMN       = win32structures.LVCOLUMNW
-            self.LVITEM         = win32structures.LVITEMW
+
+            if is64bitprocess(hwnd.process_id) or not is_x64_Python():

```

Comment:
`hwnd.process_id` should be replaced with `self.process_id()` as it was done by Igor.